sql/core/src/main/scala/org/apache/spark/sql/execution/ui/SQLAppStatusListener.scala

```diff
@@ -199,6 +199,9 @@ class SQLAppStatusListener(
 
   private def aggregateMetrics(exec: LiveExecutionData): Map[Long, String] = {
     val metricTypes = exec.metrics.map { m => (m.accumulatorId, m.metricType) }.toMap
```

Comment:
nit: this can be `val accumIds = exec.metrics.map(_. accumulatorId).toSet` now.

Reply:
Or you don't even need that; in below, where you used to do `taskMetrics.filter(m => metricTypes.contains(m._1))`, just do `taskMetrics.filter(m => metricAggregationMethods.contains(m._1))`.